README.md

```diff
@@ -1,12 +1,12 @@
 # Viceroy
 
-Viceroy provides local testing for developers working with Compute@Edge. It
-allows you to run services written against the Compute@Edge APIs on your local
+Viceroy provides local testing for developers working with Compute. It
```

Comment:
I think that this would read clearer if the first mention was "Fastly Compute" instead.